src/watertap_contrib/seto/unit_models/surrogate/tests/test_lt_med_surrogate.py

```diff
@@ -65,12 +65,25 @@ def LT_MED_frame(self):
         feed.flow_mass_phase_comp["Liq", "TDS"].fix(feed_salinity * feed_flow)
         feed.flow_mass_phase_comp["Liq", "H2O"].fix(feed_dens * feed_flow)
         feed.temperature.fix(feed_temperature + 273.15)
+        feed.pressure.fix(101325)
         steam.temperature.fix(steam_temperature + 273.15)
         # flow rate of liquid steam is zero
         steam.flow_mass_phase_comp["Liq", "H2O"].fix(0)
         dist.flow_mass_phase_comp["Liq", "TDS"].fix(0)  # salinity in distillate is zero
 
         lt_med.recovery_vol_phase[0, "Liq"].fix(recovery_ratio)
+        m.fs.water_prop.set_default_scaling(
+            "flow_mass_phase_comp", 1e-2, index=("Liq", "H2O")
+        )
+        m.fs.water_prop.set_default_scaling(
+            "flow_mass_phase_comp", 1e3, index=("Liq", "TDS")
+        )
+        m.fs.steam_prop.set_default_scaling(
+            "flow_mass_phase_comp", 1e-2, index=("Liq", "H2O")
+        )
+        m.fs.steam_prop.set_default_scaling(
+            "flow_mass_phase_comp", 1, index=("Vap", "H2O")
+        )
```

Comment:
Does the model still fail without resetting the scaling factors?

Reply:
When running a model we usually supply default scaling for the inlet mass flowrates. It might be that the case run without it, but good practice to this in general.

Reply:
I see, it makes sense for mass flowrates to fit the actual applications.